packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -59,6 +57,24 @@ Tests can be executed in any of the following supported form factors:
 | huge     | 1300  | 768    |
 | enormous | 1500  | 768    |
 
+## Environment Variables
+
+The following environment variables are configurable for the test run. Variables marked as reserved will be overwritten by the test runner. To configure reserved variables use the test runner configuration options. If the test runner is not being used the variables are configurable by the user.
+
+| Variable Name                 | Type    | Reserved | Default   | Description                                                                                                                               |
+|-------------------------------|---------|----------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| BROWSERS                      | string  | true     |           | A list of browsers for the test run. Browsers are delimited by a comma. Example: chrome,firefox,ie                                        |
+| FORM_FACTOR                   | string  | true     |           | A form factor assigned to the test run.                                                                                                   |
+| LOCALE                        | string  | true     |           | A locale assigned to the test run.                                                                                                        |
+| SELENIUM_GRID_URL             | string  | true     |           | The remote selenium grid address.                                                                                                         |
+| SITE                          | string  |          |           | A file path to a static directory of assets. When defined, an express server will launch to serve the assets and disable running webpack. |
+| THEME                         | string  | true     |           | A theme name assigned to the test run.                                                                                                    |
```

Comment:
This default value is `terra-default-theme` correct?

Reply:
Hmm, kind of. There's technically not a default if you don't use the test runner but the services do implement a default so I think it makes sense to add it here. We might need to tweak a few things to make sure this happens as expected everywhere a theme is referenced

Reply:
Added terra-default-theme as the default:

https://github.com/cerner/terra-toolkit/pull/544/commits/545286169a5e7ce61f25b37444a1105be829b750